docs/docs/widgets/date-range-picker.md

```diff
@@ -45,8 +45,8 @@ There are currently no CSA (Component-Specific Actions) implemented to regulate
 
 | <div style={{ width:"100px"}}> Variables </div> | <div style={{ width:"135px"}}> Description </div> | <div style={{ width:"135px"}}> How To Access </div> |
 |:----------- |:----------- |:--------- |
-| endDate | This variable holds the date of the endDate selected in the component. | Access the value dynamically using JS: `{{components.customcomponent1.data.title}}`. |
-| startDate | This variable holds the value assigned inside the `code` for the custom component. | Access the value dynamically using JS: `{{components.customcomponent1.data.title}}`. |
+| endDate | Holds the date selected as the end date in the component. | Accessible dynamically with JS (e.g., `{{components.customcomponent1.endDate}})`. |
```

Comment:
Hey, can you please update the query here? instead of components.customcomponent1.endDate it should be `components.daterangepicker1.endDate`. Also the ending bracket ')' should be outside the backtick.

Please do the same for startDate as well.